packages/keyring-eth-ledger-bridge/src/ledger-hw-app.ts

```diff
@@ -1,16 +1,8 @@
 import LedgerHwAppEth from '@ledgerhq/hw-app-eth';
-// eslint-disable-next-line import/no-nodejs-modules
 import { Buffer } from 'buffer';
 
 import { GetAppNameAndVersionResponse } from './type';
 
-// eslint-disable-next-line @typescript-eslint/consistent-type-definitions
-export interface MetaMaskLedgerHwAppEth extends LedgerHwAppEth {
-  openEthApp(): void;
-  closeApps(): void;
-  getAppNameAndVersion(): Promise<GetAppNameAndVersionResponse>;
-}
```

Comment:
Just confirming that this is unused

Reply:
This is indeed unused and redundant here (while also be kinda invalid 🙈)

- The `interface` is using the same name than the `class` following it, meaning the `class` will override it anyway. I believe it has been introduced to force the methods signatures on the `class` itself
- `interface` can extends `class` to some extent but it seems to have some restrictions (specifically with private members)